deploy/bootstrap/lega.sh

```diff
@@ -84,7 +84,7 @@ chroot_sessions = True
 
 [vault]
 driver = S3Storage
-url = http://s3:9000
+url = http://vault:9000
```

Comment:
The s3 -> vault changes should be placed into a separate commit.

Reply:
Maybe so, but that'll do it too